### DISCUSSION TITLE
Directus GraphQL API "Method not allowed"

### DISCUSSION THREAD

**Discussion by pacxiu** [directus/directus#2821] [Archive]
Hi,

I've just started playing with Directus and I love it. I wanted to play around with GraphQL rather than simple REST (as I need to query translations based on id from one table to another) but I can't seem to get it to work.

I could get REST API to work, set correct permissions to see properly request made to: https://cms.axieworld.com/public/Directus/items/games

I was playing on - https://lucasconstantino.github.io/graphiql-online/ but I keep  getting "Method not allowed":
![image](https://user-images.githubusercontent.com/19193499/90262291-6a45dc00-de4e-11ea-902f-6c924586161c.png)

**Comment by rijkvanzanten** [comment]
I believe the correct endpoint is `/:project/gql`, not `/` 🙂

**Reply by pacxiu** [reply]
Yes, you are right, thanks! Somehow I missed that part.
